go.mod

```diff
@@ -728,3 +729,6 @@ exclude (
 )
 
 replace github.com/github/smimesign => github.com/grafana/smimesign v0.2.1-0.20220408144937-2a5adf3481d3
+
+// Replace node_export with custom fork for multi usage.
+replace github.com/prometheus/node_exporter => github.com/grafana/node_exporter v0.18.1-grafana-r01.0.20231003194228-26e058a40f9a
```

Comment:
Could we move where the rest of replace of exporter forks are and also add the link of the PR?